### PR TITLE
Omni-channel retail recipe

### PR DESCRIPTION
**Pull request by davetroiano**
### Description

[asana](https://app.asana.com/0/1202377393885559/1202927507205735/f)

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/omnichannel_retail_recipe/
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/omnichannel_retail_recipe/omnichannel-commerce/confluent.html

### New tutorial checklist

 - [x] Implement good test cases (if relevant)
 - [x] Validate hyperlinks
 - [x] Spell check (e.g. using `aspell`)
 - [x] Full code validated in Confluent Cloud
 - [x] SQL style/syntax consistent to existing recipes
 - [x] Validate presentation with `bundle exec jekyll serve --livereload`
 - [x] Tutorial added to appropriate `index.html` or `use-case.html` file
 - [x] Source connector's auto topic naming convention works with the ksqlDB app values for `KAFKA_TOPIC` (if relevant)

### CODE REVIEW

**Comment by davetroiano** [comment]
build failure unrelated - new test case passes

**Review by bbejeck — COMMENTED** [review]
Overall looks good but there's a couple of comments about this tutorial

**Comment by bbejeck** [comment]
I see this in the tutorial 
Screen Shot 2022-10-20 at 6 01 25 PM

But I don't see any insert statements and I can't see in the source where this text is coming from

File: _includes/tutorials/omnichannel-commerce/confluent/markup/dev/manual.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+To validate that this recipe is working, run the following query:
+
+++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/omnichannel-commerce/ksql-test/code/tutorial-steps/test/validate.sql %}</code></pre>
+++++
+
+Your output should resemble:
+
+++++
```
[the remark this answers is not in the document]
Comment (davetroiano) [reply _includes/tutorials/omnichannel-commerce/confluent/markup/dev/manual.adoc]: good catch, deleted this (it's not referenced anywhere)

**Comment by davetroiano** [comment]
> I see this in the tutorial Screen Shot 2022-10-20 at 6 01 25 PM
> 
> But I don't see any insert statements and I can't see in the source where this text is coming from

good catch. This comes from shared content `shared/markup/ccloud/connect.adoc`.  For the three recipes that use datagen and no inserts (this one, campaign finance, clickstream), I removed the reference and it reads much better.

**Pull request merged**